### PR TITLE
Math magician: Fetch data from an API

### PR DESCRIPTION
**Pull request by Munyabelden**
### This PR is for

- Fetching a quote from the API Ninja and display on screen
- Added a loading and error state

**Thank you for the review** 😊

### CODE REVIEW

File: src/components/displayQoute.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from "react"
+
+function Qoute() {
+  const [data, setData] = useState([]);
+  const [loading, setLoading] = useState(false);
```
Comment (Lizdev-05) [inline src/components/displayQoute.js:5]: Great job on the implementation 👍 . According to the requirement, there should be an error state, kindly implement it. 🥺

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @Munyabelden, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
*  Descriptive Pull Request message ✔️
*  Good use of GitHub flow ✔️
*  No linter error ✔️
*  Brilliant implementation ✔️

## Required Changes ♻️
_Check the comments under the review._
- The app is currently throwing an error from the import of components which is making it unable to view the project. Kindly work on it. 🥺 
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @Munyabelden 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
- [x] Good job making most of the changes requested by the previous code reviewer. 


## Required Changes ♻️
- [ ] Dear @Munyabelden, as mentioned by the previous reviewer, the app is currently throwing an error from the import of components which is making it unable to view the project. Kindly work on it. 🥺
image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lizdev-05 — APPROVED** [review]
Hi @Munyabelden, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

-  Descriptive Pull Request message ✔️
-  Good use of GitHub flow ✔️
-  No linter error ✔️
-  Brilliant implementation ✔️
- ## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**